3p/frame.max.html

```diff
@@ -2,12 +2,14 @@
 <head>
 <meta charset="utf-8">
 <meta name="robots" content="noindex">
-<script src="./integration.js"></script>
+<script>
+var s = document.createElement('script');
```

Comment:
This might need some protection from exceptions.

Reply:
Make sense. Added.